src/SFA.DAS.FAA.Web/Views/Apply/Index.cshtml

```diff
@@ -215,7 +215,7 @@
         @if (Model.IsDisabilityConfident)
         {
             <h2 class="govuk-heading-m govuk-!-margin-top-8">Disability Confidence</h2>
-            <p class="govuk-hint">This company has joined the government's Disability Confidence scheme.</p>
+            <p class="govuk-body">This company has joined the government's Disability Confidence scheme.</p>
```

Comment:
```suggestion
            <p class="govuk-body">This company has joined the government’s Disability Confident scheme</p>
```

Reply:
The text is slightly wrong 🕺🏻

Reply:
thanks, Dan, Good spot,  I have updated it.